src/json-patch-ot-agent.js

```diff
@@ -84,10 +85,9 @@ JSONPatchOTAgent.prototype.receive = function(obj, versionedJsonPatch, applyCall
 	                    consecutivePatch,
 	                    queue.pending
 	                );
-
-	        }
-	    	apply(obj, consecutivePatch);
-		});
+			}
+			apply(this.obj, consecutivePatch);
+		}.bind(this));
```

Comment:
We are already using a closure for `queue = this` so we could use `apply(queue.obj,...` to save the performance cost of `.bind`